requests/utils.py

```diff
@@ -507,6 +507,9 @@ def get_encoding_from_headers(headers):
         # Assume UTF-8 based on RFC 4627: https://www.ietf.org/rfc/rfc4627.txt since the charset was unset
         return 'utf-8'
 
+    if 'text/csv' in content_type:
+        # Assume UTF-8 based on RFC 7111: https://tools.ietf.org/html/rfc7111 since the charset was unset
+        return 'utf-8'
 
```

Comment:
Considering that some lines to the top there is `if 'text' in content_type:`, is the herein amended code ever executed?